README.md

```diff
@@ -1,29 +1,570 @@
+
+You can read more about JSON-LD on the [JSON-LD website](jsonld).
+
+## Conformance
+
+This library aims to conform with the following:
+
+* [JSON-LD 1.0][JSON-LD 1.0], W3C Recommendation, 2014-01-16, and any [errata][errata]
+* [JSON-LD 1.0 Processing Algorithms and API][JSON-LD 1.0 API], W3C Recommendation, 2014-01-16, and any [errata][errata]
+* [JSON-LD 1.0 Framing][JSON-LD 1.0 Framing], Unofficial Draft, 2012-08-30
+* Working Group [test suite][WG test suite]
+
+The [JSON-LD Working Group][JSON-LD WG] is now developing JSON-LD 1.1. Library
+updates to conform with newer specifications will happen as features stabilize
+and development time and resources permit.
+
+* [JSON-LD 1.1][JSON-LD WG 1.1], W3C Working Draft, 2018-12-14 or [newer][JSON-LD WG latest]
+* [JSON-LD 1.1 Processing Algorithms and API][JSON-LD WG 1.1 API], W3C Working Draft, 2018-12-14 or [newer][JSON-LD WG API latest]
+* [JSON-LD 1.1 Framing][JSON-LD WG 1.1 Framing], W3C Working Draft, 2018-12-14 or [newer][JSON-LD WG Framing latest]
+
+The [test runner][] is often updated to note or skip newer tests that are not
+yet supported.
 
 ## Supported frameworks
 
 * .NET 4.0
 * .NET Standard 1.3 and 2.0
 * Portable .NET 4.5, Windows 8
 
+## Installation
+
+### dotnet CLI
+
+```sh
+dotnet new console
+dotnet add package json-ld.net
+```
+
+```csharp
+using JsonLD.Core;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace JsonLD.Demo
+{
+    internal class Program
+    {
+        private static void Main()
+        {
+            var json = "{'@context':{'test':'http://www.test.com/'},'test:hello':'world'}";
+            var document = JObject.Parse(json);
+            var expanded = JsonLdProcessor.Expand(document);
+            Console.WriteLine(expanded);
+        }
+    }
+}
+
+```
+
+## Examples
+--------
+
+Example data and context used throughout examples below:
+
+#### doc.json
+```json
+{
+    "@id": "http://example.org/ld-experts",
+    "http://schema.org/name": "LD Experts",
+    "http://schema.org/member": [{
+        "@type": "http://schema.org/Person",
+        "http://schema.org/name": "Manu Sporny",
+        "http://schema.org/url": {"@id": "http://manu.sporny.org/"},
+        "http://schema.org/image": {"@id": "http://manu.sporny.org/images/manu.png"}
+    }]
+}
+```
+
+#### context.json
+```json
+{
+    "name": "http://schema.org/name",
+    "member": "http://schema.org/member",
+    "homepage": {"@id": "http://schema.org/url", "@type": "@id"},
+    "image": {"@id": "http://schema.org/image", "@type": "@id"},
+    "Person": "http://schema.org/Person",
+    "@vocab": "http://example.org/",
+    "@base": "http://example.org/"
+}
+```
+
+
+### [compact](http://json-ld.org/spec/latest/json-ld/#compacted-document-form)
```

Comment:
Headers should preferably not link to somewhere else, as they are themselves anchors within this document. Please add a short paragraph explaining what compaction is that links to the external reference.

```suggestion
### Compact

[Compact](http://json-ld.org/spec/latest/json-ld/#compacted-document-form).
```

Reply:
This convention of linking from the headers, the lack of prose documentation, and (almost) all documentation which _is_ included is lifted from the pseudo-canonical json-ld.js repo. My hope was that we could stick fairly closely to that pattern. However, maybe I could lift it from the web standard or the main json-ld website.